### PR TITLE
feat: add .station

### PR DESCRIPTION
**Pull request by benw202**
## Description

Similar to .metar but gives information about the airport instead

## Test Results

image

## Discord Username

BenW#8484

### CODE REVIEW

File: src/commands/utils/station.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            msg.channel.send(makeEmbed({
+                title: `Station info | ${stationReport.icao}`,
+                description: makeLines([
+                    '**Station Information:**',
+                    `**Name:** ${stationReport.name}`,
+                    `**Country:** ${stationReport.country}`,
+                    `**City:** ${stationReport.city}`,
+                    `**Latitude:** ${stationReport.latitude}`,
+                    `**longitude:** ${stationReport.longitude}`,
+                    ,
```
Comment (GhostEagle68) [inline src/commands/utils/station.ts:34]: What about adding 
`**elevation_ft:** ${stationReport.elevation_ft:}`,
`**elevation_m:** ${stationReport.elevation_m:}`,
to show the airports' elevation in feet and meters?

Comment (benw202) [reply src/commands/utils/station.ts:34]: @GhostEagle68 These will be added in, this command is still in progress while we work out a few things!

**Comment by NUTTTTTTTT** [comment]
LVVGTM!

**Comment by tracernz** [comment]
Could we have it give runways in metres outside of the USA, or dual unit worldwide? It's hard to visualize what 10500 people's feet end-to-end look like, whereas I can easily visualise the distance light travels in 10 microseconds. 😆

**Pull request merged**